p2panda-rs/src/message/message_encoded.rs

```diff
@@ -110,18 +110,19 @@ mod tests {
         assert!(MessageEncoded::new("68656c6c6f2062616d626f6f21").is_err());
 
         // Valid `delete` message
-        assert!(MessageEncoded::new("a466616374696f6e6664656c65746566736368656d6178843030343032646332356433326466623430306262323935623636336434373036626334376630636234663165646666323737633733376166633861393233323333306165393838346663663664303231343161373835633566643832633139366239373365383432376566633063303464303434346463633330353932323062396564616776657273696f6e016269647884303034306366393466366436303536353765393063353433623063393139303730636461616637323039633565316561353861636238663335363866613231313432363864633961633362616665313261663237376432383666636537646335396237633063333438393733633465396461636265373934383565353661633261373032").is_ok());
+        assert!(MessageEncoded::new("a466616374696f6e6663726561746566736368656d61784430303230623137376563316266323664666233623730313064343733653664343437313362323962373635623939633665363065636266616537343264653439363534336776657273696f6e02666669656c6473a168757365726e616d65a26474797065637374726576616c75656462756275").is_ok());
```

Comment:
So short!!! :heart:

Reply:
.. wait until we have encrypted fields with padding inside :sob: :rofl:

Reply:
booooooooo!